.github/workflows/lint.yml

```diff
@@ -0,0 +1,20 @@
+name: Rust Format Check
```

Comment:
Just realized this file should be `fmt.yml` instead of `lint.yml`. Linting is in `ci.yml`.